runtest.py

```diff
@@ -88,10 +90,17 @@ def clean_up_output_data(self):
             shutil.rmtree(output_dir)
 
     def _load_settings(self):
+        if not self.firefox_settings:
+            return {}
         try:
             if os.path.exists(self.firefox_settings):
                 with open(self.firefox_settings) as firefox_settings_fh:
-                    return json.load(firefox_settings_fh)
+                    settings = json.load(firefox_settings_fh)
+                    self.logger.info('** Important **\n'
```

Comment:
well, I already wait few seconds for notice people.
I think we put the message to logger, and logger will handle the message base on it's format string.

If we want to show larger warning message like:
```
##########
# WARNING #
##########
some message...
```
we have to put `\n` in the beginning of message, that will break the logger's format.